CHANGELOG.md

```diff
@@ -26,10 +26,13 @@ All notable changes to this project will be documented in this file.
 - Document that Spark Connect doesn't integrate with the history server ([#559])
 - test: Bump to Vector `0.46.1` ([#565]).
 - Use versioned common structs ([#572]).
+- BREAKING: Change the label `app.kubernetes.io/name` for Spark history and connect objects to use `spark-history` and `spark-connect` instead of `spark-k8s` ([#573]).
+- BREAKING: The history  pods now have their own `ClusterRole` named `spark-history-clusterrole` ([#573]).
```

Comment:
According to https://docs.stackable.tech/home/stable/contributor/docs/style-guide/#_formatting_kubernetes_style_guide
```suggestion
- BREAKING: The history Pods now have their own ClusterRole named `spark-history-clusterrole` ([#573]).
```